source/exploring_catalog_and_datasets/02_looking_up_a_dataset/looking_up_dataset.rst

```diff
@@ -0,0 +1,39 @@
+Looking up a dataset
+====================
+
+Once the right dataset has been found in the catalog, through the searching and filtering options or by scrolling down, it should be time to look it up. The visualization interface of a dataset is the same, no matter the portal.
+
+.. image:: images/interface_lookup.png
+   :alt: Interface of chosen dataset, with filters and exploration tabs
+
+On the left side are displayed:
```

Comment:
I guess its nitpicking but this phrase sounds a bit weird to me. 

Could be "Displayed on the left side is" or "On the left side there is"